prime-dotnet-webapi/Services/EnrolleePaperSubmissionService.cs

```diff
@@ -264,5 +264,119 @@ public async Task FinalizeSubmissionAsync(int enrolleeId)
             _context.Set<T>().RemoveRange(oldItems);
             _context.Set<T>().AddRange(itemList);
         }
+
+        public async Task<bool> IsPotentialPaperEnrolleeReturnee(DateTime dateOfBirth)
+        {
+            return await _context.Enrollees
+                .AsNoTracking()
+                .AnyAsync(
+                    e => e.GPID.StartsWith(PaperGpidPrefix)
+                    && e.DateOfBirth.Date == dateOfBirth.Date
+                    && !_context.EnrolleeLinkedEnrolment.Any(
+                        ele => ele.IsConfirmed
```

Comment:
This line, unfortunately, does not do what you are hoping. 
```c#
_context.EnrolleeLinkedEnrolment.Any(ele => ele.IsConfirmed)
```
asks the database if there are any Linked Enrollments that are confirmed, regardless of who owns them.

Reply:
Ops that was supposed to check the against the enrolleeId 😄